samples/bluetooth/central_bas/sample.yaml

```diff
@@ -1,9 +1,9 @@
 sample:
-  description: TBD
-  name: TBD
+  name: BLE Central BAS
+  description: The bluetooth low energy central battery service application
```

Comment:
Remove "The" and use capital letters: _Bluetooth Low Energy_ for consistency with other descriptions.

Reply:
Ok removed